internal/controllers/rest_controller.go

```diff
@@ -59,10 +119,10 @@ func (rc RESTController) OIDCProviderCallback(w http.ResponseWriter, r *http.Req
 	codeProvider := r.URL.Query().Get("code")
 
 	// Creates an OAuth2 token with the auth code returned from the OIDC Provider.
-	token, err := rc.oidcService.ClaimsToken(ctx, codeProvider)
+	token, err := rc.oidcService.Claims(ctx, codeProvider)
 	if err != nil {
 		rc.logger.Error(err)
```

Comment:
same as above

Reply:
we also already have a custom error coming from Claims